masterlist.yaml

```diff
@@ -125,6 +122,8 @@ plugins:
   - name: 'Anchorage.esm'
     url: [ 'http://taleoftwowastelands.com/']
     priority: -400998000
+    after:
+      - 'Fallout3.esm'
```

Comment:
Tale of Two Wastelands merges Fallout3.esm contents (as well as all Fo3 DLCs) into Fallout: New Vegas